cypress/features/regression/buttonToggle.feature

```diff
@@ -84,3 +81,9 @@ Feature: Button Toggle component
       And I uncheck grouped checkbox
       And I enable Button component
     Then Button Toggle is not grouped
+
+  @positive
+  Scenario: Verify the onChange function for a Button Toggle component
+    Given clear all actions in Actions Tab
+    When I click on Button Toggle
```

Comment:
In this case you will check only onChange event for the last button in group as you have 3 clicks on every button

Reply:
Amended to click on then check each button in a separate scenario example 👍